ext/debian/pxp-agent.init

```diff
@@ -0,0 +1,113 @@
+#! /bin/sh
+### BEGIN INIT INFO
+# Provides:          pxp-agent
+# Required-Start:    $network $named $remote_fs $syslog
+# Required-Stop:     $network $named $remote_fs $syslog
+# Should-Start:      pxp-agent
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+### END INIT INFO
+
+PATH=/opt/puppetlabs/puppet/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+
+exec=/opt/puppetlabs/puppet/bin/pxp-agent
+prog="pxp-agent"
+desc="PXP agent"
+
+pidfile="/var/run/puppetlabs/${prog}.pid"
+
+[ -x $exec ] || exit 5
+
+[ -r /etc/default/pxp-agent ] && . /etc/default/pxp-agent
+
+. /lib/lsb/init-functions
+
+reload_pxp_agent() {
+    start-stop-daemon --stop --quiet --signal HUP --pidfile $pidfile --name $prog
```

Comment:
`signal(SIGHUP, SIG_IGN);`

So HUP is currently ignored and marked as a TODO in the daemonize code. I think it is misleading to offer a reload method that we know won't work. Maybe leave the function, but drop it out of the list at the end?